src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+---
+linkTitle: Cluster API architecture
+title: The Giant Swarm Platform Architecture
```

Comment:
This title should differentiate, as we have some other architecture docs/versions, too.

Also, please use sentence case.

```suggestion
title: The Giant Swarm platform architecture using Cluster API
```